### PR TITLE
migration modal not to show same region again

### PR DESCRIPTION
**Pull request by WangLiNaruto**
Fixed eclipse-xpanse/xpanse#1579
image
image
image
image
image
image

### CODE REVIEW

**Comment by Alice1319** [comment]
LGTM

File: src/components/content/order/migrate/SelectMigrateTarget.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    DeployedServiceDetails,
+    UserOrderableServiceVo,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../xpanse-api/generated';
+import { getAvailableServiceHostingTypes } from '../formDataHelpers/serviceHostingTypeHelper';
+import { Tab } from 'rc-tabs/lib/interface';
+import { convertAreasToTabs } from '../formDataHelpers/areaHelper';
+import { getRegionDropDownValues } from '../formDataHelpers/regionHelper';
+import { Regions } from '../types/Regions';
+export const SelectMigrateTarget = ({
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectMigrateTarget.tsx:19]: the component name can be `SelectMigrationTarget`

File: src/components/content/order/types/Regions.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@
 /**
  * The regions inside Area
  */
-export interface Region {
+export interface Regions {
```
Comment (swaroopar) [inline src/components/content/order/types/Regions.ts:9]: this is still a single `Region` right? or does this type store multiple regions always?

File: src/components/content/order/migrate/SelectMigrateTarget.tsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    <div className={'order-select-migrate-target'}>
+                        <div className={'order-param-item-left'} />
+                        <div className={'order-param-item-content'}>
+                            <Form.Item
+                                name='Change Region/Cloud'
+                                label='Change Regions/Cloud'
+                                rules={[
+                                    {
+                                        required: true,
+                                        message: 'Please select migration between regions or migration between clouds',
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectMigrateTarget.tsx:142]: Please select the type of migration you wish to perform.

File: src/components/content/order/migrate/SelectMigrateTarget.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+    return (
+        <>
+            <>
+                <Form form={form} layout='vertical' autoComplete='off' onFinish={next} validateTrigger={['next']}>
+                    <div className={'order-select-migrate-target'}>
+                        <div className={'order-param-item-left'} />
+                        <div className={'order-param-item-content'}>
+                            <Form.Item
+                                name='Change Region/Cloud'
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectMigrateTarget.tsx:137]: Choose the migration Destination

File: src/components/content/order/migrate/SelectMigrateTarget.tsx [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+                                label='Change Regions/Cloud'
+                                rules={[
+                                    {
+                                        required: true,
+                                        message: 'Please select migration between regions or migration between clouds',
+                                    },
+                                ]}
+                            >
+                                <Radio.Group onChange={onChange} value={target}>
+                                    <Radio value={'region'}>Regions</Radio>
```
Comment (swaroopar) [inline src/components/content/order/migrate/SelectMigrateTarget.tsx:147]: Text for both the buttons should be ->
- Migrate service to a different region offered by the same cloud service provider. 
- Migrate service to a different cloud service provider.

Also put the options one below the other.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**